ml_git/storages/multihash_storage.py

```diff
@@ -27,5 +27,5 @@ def check_integrity(self, cid, ncid):
         if cid == ncid:
             log.debug(output_messages['DEBUG_CHECKSUM_VERIFIED'] % cid, class_name=MULTI_HASH_STORAGE_NAME)
             return True
-        log.error(output_messages['ERROR_CORRUPTION_DETECTED'] % (cid, ncid), class_name=MULTI_HASH_STORAGE_NAME)
+        log.debug(output_messages['ERROR_CORRUPTION_DETECTED'] % (cid, ncid), class_name=MULTI_HASH_STORAGE_NAME)
```

Comment:
```suggestion
        log.debug(output_messages['DEBUG_CORRUPTION_DETECTED'] % (cid, ncid), class_name=MULTI_HASH_STORAGE_NAME)
```
If you adopt this suggestion, please remember to update the `ml_git_message.py`.

Reply:
Updated at https://github.com/HPInc/ml-git/pull/206/commits/3f4defdec38cf4e311cb3a2ae2104a51355d132f